lib/matplotlib/tests/test_font_manager.py

```diff
@@ -23,3 +24,16 @@ def test_font_priority():
     cmap = font.get_charmap()
     assert len(cmap) == 131
     assert cmap[8729] == 30
+
+
+def test_otf():
+    fname = '/usr/share/fonts/opentype/freefont/FreeMono.otf'
+    if os.path.exists(fname):
+        assert is_opentype_cff_font(fname)
+        assert False
```

Comment:
This is in here to make sure that this file really exists on travis, once I see it fail on this assert I will remove the test.